README

```diff
@@ -145,7 +142,13 @@ REQUIREMENTS:
         libpython (optional)
         libidn2 or libidn (optional)
         elfutils >= 158 (optional)
-        make, gcc, and similar tools
+        pkg-config
+        gperf >= 3.1
+        docbook-xsl (optional, required for documentation)
+        xsltproc    (optional, required for documentation)
+        python-lxml (optional, required to build the indices)
+        python, meson, ninja
+        gcc, awk, sed, and similar tools
```

Comment:
Should m4 be added here?
With autotools this was sort of implied, but with meson it's probably less common to require m4.